Resources/Public/Javascript/OpenLayers/get-ol.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+VERSION="6.12.0"
+
+wget -O ol.zip "https://github.com/openlayers/openlayers/releases/download/v$VERSION/v$VERSION-dist.zip"
+unzip -p ol.zip "v$VERSION-dist/ol.js" > openlayers.js
+unzip -p ol.zip "v$VERSION-dist/ol.css" > openlayers.css
```

Comment:
Before, we made a custom build of the Openlayers library. Isn't this still useful or is this not possible anymore?
Now the full openlayers.js is about 1,3 MB, the custom build before 190 kB.

The transfered data to the browser in the workview is only about 1,7 MB (including image, libraries, HTML) vs. 1,5 MB before. This is due to compression on the transfer, I think. So it's usable. But of course, if we could reduce the size here, it could be worth maybe.

Reply:
I see, I had a remark on this but it's buried in the commit notes.

So I think it would still be useful to reduce the bundle size considering that parsing the decompressed JS also takes time. My idea was to postpone this until we start using a bundler (something like Webpack) for our JS.

As far as I see, OL doesn't anymore have a build tool as before where features could be selected in a configuration file. If you think it's urgent, it would probably be possible to do the build manually by writing a JS file that exports the necessary components.

The approach in OL6 seems to be to use ES6 import together with a bundler. That is, we wouldn't rely on a global openlayers.js anymore, but only import what we need in the application. Then Webpack, via "tree-shaking", would only include the code that is actually used in the final bundle. That way we also wouldn't need to know and list what we use in a separate file.

Reply:
Ok, thank you. I haven't read your remark about this.

Yes, you can postpone this point. I only wanted to be sure, you kept it in mind. 

Actually, the browser loads the JavaScript only once and loads it from cache afterwards. Of course the parsing time has to be done on every load (as long the browser doesn't cache it either). So I could live with this and maybe we get better later on with webpack or something similar.